src/Modules/Circuits/circuit.css

```diff
@@ -0,0 +1,32 @@
+
+.circuitslist {
+  margin-top: 130px;
+  display: flex;
+  flex-wrap: wrap;
+  width: 84vw;
+  margin-inline: 8vw;
+}
```

Comment:
- [ ] One of the project requirements is to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements

I have noticed that your design some elements are different:
1. Please add the font family as per design:
<img width="706" alt="Screenshot 2022-10-06 at 20 25 27" src="https://user-images.githubusercontent.com/77038610/194378996-bd343e18-069a-459b-b945-b0189f002bd0.png">

- [ ] OPTIONAL The Application on design guidelines is different than yours, you have 3 menus like drivers, circuits and standings and each has view, but when I click on particular circuit I don't see any stats, as a user I'm confused why circuit is there (sorry I have no idea of races hahah). As a user I also have no idea why `Standings` is there in the menu. When I click on drivers I can see 2022 empty stats, also not clear is it stat for driver, or stat overall for all drivers etc. And sometimes this stat is empty: 
<img width="892" alt="Screenshot 2022-10-06 at 20 45 38" src="https://user-images.githubusercontent.com/77038610/194382848-82fe64bc-59db-474e-ab5a-20fb5d409f04.png">

The app you were supposed to build should have many categories, when you click on one, subcategories were supposed to appear like country and cities or family and members. I understand that if I make this requirement required, you would have to rewrite everything and build the whole app again. I just want to strongly recommend you, to plan the task you want to work on  😃 

Overall you have done a great job, properly setup the redux and I can see that you really enjoyed what you built which is the most important :)

- [ ] OPTIONAL: I would also recommend to work on styles better, capstone projects are perfect show cases for future job hunt! However to show that you have brilliant codebase, you should be able to show a beautiful page. I will just pint out some design issues:

Header is too high, because the menu is vertical, should be horisontal. Menu items are row they should be bigger and better styled. 
<img width="1422" alt="Screenshot 2022-10-06 at 20 35 28" src="https://user-images.githubusercontent.com/77038610/194380930-5c49fe61-3c99-4e14-ab36-3ff9d69edffe.png">

The font color and page has no contrast at some places it becomes impossible to read the content:
<img width="992" alt="Screenshot 2022-10-06 at 20 37 04" src="https://user-images.githubusercontent.com/77038610/194381257-02888063-c7a5-425f-847d-09f30a3fa771.png">

On homepage go back button is visible but it does nothing, it would be nice to style it too by using libraries currently it is just '<'

<img width="54" alt="Screenshot 2022-10-06 at 20 39 13" src="https://user-images.githubusercontent.com/77038610/194381683-03a0f402-c0c3-4a93-bdf0-e4fbaaf2b3df.png">

Reply:
Thank you, I noticed that the API does take some time to load the Stats and it will only show for the 2022 lineup, the Items on the navmenu on a column I did it on purpose, I like how it looks but I will take your advice, Thank You.